### PR TITLE
Update QUEST.tsv

### PR DESCRIPTION
**Pull request by wlghksdl2**
Added Translations 002275 to 002295
EXCEPT 002276, 002277 and 002291
Added Translations 2306 - 2312. Please take a look. Thanks.
Added 2313-2326. Please take a look. Almost done Aiste's story.
Added upto 2331. Finished Aiste's story.

### CODE REVIEW

File: QUEST.tsv [hunk 1]
```diff
@@ -2272,24 +2272,24 @@ QUEST_20150717_002271	[Usage Limit]{nl}- Due to a recent incident, we sealed the
...
-QUEST_20150729_002288
-QUEST_20150729_002289
-QUEST_20150729_002290
-QUEST_20150729_002291
-QUEST_20150729_002292
-QUEST_20150729_002293
-QUEST_20150729_002294
-QUEST_20150729_002295
+QUEST_20150729_002281 Hm.. Have the party members fully gathered yet?
+QUEST_20150729_002282 Welcome to the Mercenary Quest Office. {nl}But.. There aren't appropriate requests yet.{nl}
```
Comment (emailboxu) [inline QUEST.tsv]: "There aren't any appropriate"

File: QUEST.tsv [hunk 2]
```diff
@@ -2272,24 +2272,24 @@ QUEST_20150717_002271	[Usage Limit]{nl}- Due to a recent incident, we sealed the
...
-QUEST_20150729_002289
-QUEST_20150729_002290
-QUEST_20150729_002291
-QUEST_20150729_002292
-QUEST_20150729_002293
-QUEST_20150729_002294
-QUEST_20150729_002295
+QUEST_20150729_002281 Hm.. Have the party members fully gathered yet?
+QUEST_20150729_002282 Welcome to the Mercenary Quest Office. {nl}But.. There aren't appropriate requests yet.{nl} 
+QUEST_20150729_002283 When you return with reliable comrades, when I will give you an appropriate quest.{nl}If there is a request.{nl}
```
Comment (emailboxu) [inline QUEST.tsv]: "I will give you an appropriate quest when you return with reliable comrades.{n1} ...If there are such requests."

File: QUEST.tsv [hunk 3]
```diff
@@ -2272,24 +2272,24 @@ QUEST_20150717_002271	[Usage Limit]{nl}- Due to a recent incident, we sealed the
...
-QUEST_20150729_002291
-QUEST_20150729_002292
-QUEST_20150729_002293
-QUEST_20150729_002294
-QUEST_20150729_002295
+QUEST_20150729_002281 Hm.. Have the party members fully gathered yet?
+QUEST_20150729_002282 Welcome to the Mercenary Quest Office. {nl}But.. There aren't appropriate requests yet.{nl} 
+QUEST_20150729_002283 When you return with reliable comrades, when I will give you an appropriate quest.{nl}If there is a request.{nl} 
+QUEST_20150729_002284 There is a good request that just came in.{nl}Do you want to take it?
+QUEST_20150729_002285 Anyway, that is very strange.{nl}If it was Goddess Laima's will, not anyone could undo her seal.
```
Comment (emailboxu) [inline QUEST.tsv]: "will, no one could undo her seal."

File: QUEST.tsv [hunk 4]
```diff
@@ -2272,24 +2272,24 @@ QUEST_20150717_002271	[Usage Limit]{nl}- Due to a recent incident, we sealed the
...
+QUEST_20150729_002286 Whether this revelation will become a new beacon of hope or a start of a new disaster, it is indeed difficult to judge.. {nl}We believe we will continue to investigate.
```
Comment (emailboxu) [inline QUEST.tsv]: "We are thinking of continuing our investigation."

File: QUEST.tsv [hunk 5]
```diff
@@ -2272,24 +2272,24 @@ QUEST_20150717_002271	[Usage Limit]{nl}- Due to a recent incident, we sealed the
...
-QUEST_20150729_002294
-QUEST_20150729_002295
+QUEST_20150729_002281 Hm.. Have the party members fully gathered yet?
+QUEST_20150729_002282 Welcome to the Mercenary Quest Office. {nl}But.. There aren't appropriate requests yet.{nl} 
+QUEST_20150729_002283 When you return with reliable comrades, when I will give you an appropriate quest.{nl}If there is a request.{nl} 
+QUEST_20150729_002284 There is a good request that just came in.{nl}Do you want to take it?
+QUEST_20150729_002285 Anyway, that is very strange.{nl}If it was Goddess Laima's will, not anyone could undo her seal.
+QUEST_20150729_002286 Whether this revelation will become a new beacon of hope or a start of a new disaster, it is indeed difficult to judge.. {nl}We believe we will continue to investigate. 
+QUEST_20150729_002287 )}My son volunteered to join the army.{nl}So I have been offering my prayers to the greatest Goddess Ausrine.{nl}
+QUEST_20150729_002288 )}Monsters are keep multiplying in number to the point the people cannot safely live in the forest anymore.{nl}At least Klaipeda is still safe.{nl}
```
Comment (emailboxu) [inline QUEST.tsv]: "Monsters have been multiplying to the point where people cannot safely live in the forest anymore."

File: QUEST.tsv [hunk 6]
```diff
@@ -2272,24 +2272,24 @@ QUEST_20150717_002271	[Usage Limit]{nl}- Due to a recent incident, we sealed the
...
+QUEST_20150729_002289 Maybe the Goddess Ausrine gave us her protection.{nl}Isn't she the Goddess who leads all other Goddesses?
+QUEST_20150729_002290 )}I am glad my son dreams to become a soldier..{nl}But the monsters are keep increasing in number, I am just worried he will be wounded.
```
Comment (emailboxu) [inline QUEST.tsv]: "...dreams of becoming a soldier..{n1}But the monsters keep increasing in number, and I'm worried he will be injured."

File: QUEST.tsv [hunk 7]
```diff
@@ -2272,24 +2272,24 @@ QUEST_20150717_002271	[Usage Limit]{nl}- Due to a recent incident, we sealed the
...
+QUEST_20150729_002291 
+QUEST_20150729_002292 After the night passes, doesn't the morning dawn? The Goddess Ausrine is the one who brings us the morn. {nl}Not only that, but when we die we go to the Goddess Ausrine.
```
Comment (emailboxu) [inline QUEST.tsv]: "Doesn't the morning come after the night? ...the one who brings us the morning."

File: QUEST.tsv [hunk 8]
```diff
@@ -2272,24 +2272,24 @@ QUEST_20150717_002271	[Usage Limit]{nl}- Due to a recent incident, we sealed the
...
+QUEST_20150729_002293 Eternal life..{nl}Through the Goddess Ausrine's power my life extends for every owl statue I sculpt.{nl}If this old body of mine can give her strength and help the world then I am determined to carry on forever. 
+QUEST_20150729_002294 Are you interested in life everlasting?{nl}Since necromancers deal with the dead, they can feel life force and depending on their ability, manipulate it.{nl}If you are as good as me, absorbing that life energy to extend life is also possible.
```
Comment (emailboxu) [inline QUEST.tsv]: "...dead, they can sense life force, and, depending..."

"If you are as good as I am, you can even absorb that life force to extend your own."

File: QUEST.tsv [hunk 9]
```diff
@@ -2272,24 +2272,24 @@ QUEST_20150717_002271	[Usage Limit]{nl}- Due to a recent incident, we sealed the
...
+QUEST_20150729_002295 Are you curious how I obtained eternal life?{nl}If stilled time is eternal, then you can consider it eternal life.{nl}By stopping time I maintain a body that is neither aging or dying.{nl}Whether or not quiescence can be expressed as eternal, I do not know.
```
Comment (emailboxu) [inline QUEST.tsv]: "Are you curious as to how I obtained..."

File: QUEST.tsv (hunk 1 above)
Comment (wlghksdl2) [inline QUEST.tsv]: I don't like "Mercenary Quest Office" 용병 의뢰소 translation. What do you think is more appropriate?

File: QUEST.tsv (hunk 2 above)
Comment (wlghksdl2) [inline QUEST.tsv]: Good catch, this was a bad on my part

File: QUEST.tsv (hunk 3 above)
Comment (wlghksdl2) [inline QUEST.tsv]: Original text implied undoing her seal is a very difficult task but not impossible. 
I still don't like how my translation flows, I will come back to it later

File: QUEST.tsv (hunk 4 above)
Comment (wlghksdl2) [inline QUEST.tsv]: Ok, good edit thanks

File: QUEST.tsv (hunk 5 above)
Comment (wlghksdl2) [inline QUEST.tsv]: Good edit. Thanks

File: QUEST.tsv (hunk 1 above)
Comment (emailboxu) [inline QUEST.tsv]: Quest is probably the shortest word you're going to find for that. Commissioner's Office or Missions Office would work too.

File: QUEST.tsv (hunk 6 above)
Comment (wlghksdl2) [inline QUEST.tsv]: Good edit thanks

File: QUEST.tsv (hunk 7 above)
Comment (wlghksdl2) [inline QUEST.tsv]: Sounds better, thanks

File: QUEST.tsv (hunk 3 above)
Comment (emailboxu) [inline QUEST.tsv]: "not just anyone" works here then.

File: QUEST.tsv (hunk 9 above)
Comment (wlghksdl2) [inline QUEST.tsv]: Thanks for the edit. Much better.

File: QUEST.tsv (hunk 1 above)
Comment (wlghksdl2) [inline QUEST.tsv]: Commissioner's Office seems to fit pretty well.

**Comment by wlghksdl2** [comment]
Added more translations, if anyone could look over and comment that would help.
Thanks.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**